.gitignore

```diff
@@ -2,4 +2,4 @@ protos/**/*.cache
 worlds/.*.wbproj
 build/
 
-nugus_controller
+/controllers/nugus_controller/nugus_controller
```

Comment:
This is wrong, unless you also fix the [copy command](https://github.com/NUbots/Webots/blob/a230130732e5cdf61f4c40a9fd943bf00bc31a7f/controllers/nugus_controller/CMakeLists.txt#L14-L16)

Reply:
Ah bother you're right. I have too many files lying around from all these PRs now :sweat_smile:  
I'll open a PR for fixing the copy command and get that merged first then

Reply:
You can also change `${CMAKE_CURRENT_BINARY_DIR}/nugus_controller` to `$<TARGET_FILE:nugus_controller>` which will resolve to the full path to the file generated by the `nugus_controller` target
```
add_custom_command(TARGET nugus_controller POST_BUILD COMMAND ${CMAKE_COMMAND} -E
  copy "$<TARGET_FILE:nugus_controller>" "${CMAKE_CURRENT_SOURCE_DIR}/nugus_controller"
)
```